beaker.gemspec

```diff
@@ -33,7 +33,7 @@ Gem::Specification.new do |s|
   # Run time dependencies
   s.add_runtime_dependency 'minitest', '~> 5.4'
   s.add_runtime_dependency 'minitar', '~> 0.5.4'
-  s.add_runtime_dependency 'json', '~> 1.8'
+  s.add_runtime_dependency 'json', '~> 1.4'
```

Comment:
After some more reading / experimenting, I think dropping is the best bet.

However, I'm still not getting the desired behavior in my testing with this change.  Bundler is supposed to be OK with using a mix of system gems and local gems (isolated with `--path`) as long as you are not using its `--deployment` switch.

However, in testing with this combination of gems I'm still seeing Bundler try to pull from rubygems when the local system gem satisfies the dependency requirements.

I've tried explicitly specifying `= 1.5.5` when on Ruby 1.9.3 on Windows *and* I've tried omitting the `json` dependency altogether.  In both cases, it contacts `rubygems` to acquire gem downloads, and will pull one.  When `= 1.5.5` is specified, it downloads it despite having the same version installed as a system gem.  When omitted, it will download `1.8.3`.

`.bundle/config` contains a variable `BUNDLE_DISABLE_SHARED_GEMS` which is supposed to influence Bundlers behavior - but I can't seem to override it at all (through environment variable or command line switch).  If I edit that file to allow shared gems (i.e. the system ones), then proceed to call `bundle install --path .bundle/gems` it reverts the value.  So I'm wondering if bundler is just buggy here?

Note that I have seen the desired behavior when simply using `gem install` (like for the azure sdk gem, which previously had similar problems)